Firestore/Example/Tests/Local/FSTLRUGarbageCollectorTests.mm

```diff
@@ -42,13 +42,44 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+struct TestResources {
+  id<FSTPersistence> persistence;
+  id<FSTQueryCache> queryCache;
+  // TODO(gsoltis): other tables as necessary
+  FSTListenSequenceNumber initialSequenceNumber;
+};
+
 @implementation FSTLRUGarbageCollectorTests {
   FSTTargetID _previousTargetID;
   int _previousDocNum;
   FSTObjectValue *_testValue;
   FSTObjectValue *_bigObjectValue;
 }
 
+- (TestResources)allocateResources {
+  id<FSTPersistence> persistence = [self newPersistence];
```

Comment:
I did it this way because of one test that creates a new `persistence` in a loop. Maybe I just need to separate out those test cases or something though.